com.unity.render-pipelines.universal/Shaders/BRGPicking.shader

```diff
@@ -0,0 +1,75 @@
+Shader "Hidden/Universal Render Pipeline/BRGPicking"
+{
+    SubShader
+    {
+        // Universal Pipeline tag is required. If Universal render pipeline is not set in the graphics settings
+        // this Subshader will fail. One can add a subshader below or fallback to Standard built-in to make this
+        // material work with both Universal Render Pipeline and Builtin Unity Pipeline
+        Tags{"RenderType" = "Opaque" "RenderPipeline" = "UniversalPipeline" "UniversalMaterialType" = "Lit" "IgnoreProjector" = "True" "ShaderModel"="4.5"}
+        LOD 300
+
+        Pass
+        {
+            Name "ScenePickingPass"
+            Tags { "LightMode" = "Picking" }
+
+            Cull [_CullMode]
+
+            HLSLPROGRAM
+
+            #pragma target 4.5
+            #pragma only_renderers d3d11 playstation xboxone xboxseries vulkan metal switch
+
+            #pragma editor_sync_compilation
+            #pragma multi_compile DOTS_INSTANCING_ON
+
+            #pragma vertex Vert
+            #pragma fragment Frag
+
+            #include "Packages/com.unity.render-pipelines.universal/ShaderLibrary/Core.hlsl"
+
+            float4x4 unity_BRGPickingViewMatrix;
+            float4x4 unity_BRGPickingProjMatrix;
+            float4 unity_BRGPickingSelectionID;
+
+            struct Attributes
+            {
+                float4 positionOS : POSITION;
+                UNITY_VERTEX_INPUT_INSTANCE_ID
+            };
+
+            struct Varyings
+            {
+                float4 positionCS : SV_POSITION;
+                UNITY_VERTEX_INPUT_INSTANCE_ID
+                UNITY_VERTEX_OUTPUT_STEREO
+            };
+
+            Varyings Vert(Attributes input)
+            {
+                Varyings output = (Varyings)0;
+                UNITY_SETUP_INSTANCE_ID(input);
+                UNITY_TRANSFER_INSTANCE_ID(input, output);
+                UNITY_INITIALIZE_VERTEX_OUTPUT_STEREO(output);
+
+                float4 positionWS = mul(UNITY_MATRIX_M, input.positionOS);
```

Comment:
This dance is needed with HDRP because of camera relative rendering. Camera relative rendering is not needed for the picking and causes issues because it uses the `_WorldSpaceCameraPos` constant, which is affected by the interference problem I described in my previous comment https://github.com/Unity-Technologies/Graphics/pull/6071#discussion_r745587459